### DISCUSSION TITLE
Related to Billing cycle

### DISCUSSION THREAD

**Discussion by bhumzz18101999** [Kyon147/laravel-shopify#197] [Q&A]
As per this package the charge is generated using the following route : route('billing', ['plan' => 2, 'shop' => $shopDomain])

So my question is whenever any paid plan store is charged or any development store is charged every time in charges table the "test" column is filled with "1".

Can you please give me a brief explanation about how this charge system works?

Actually, I want to integrate partner friendly concept in my shopify app.

**Comment by RishabhTayal** [comment]
Same. Any answers yet?
